### PR TITLE
Update: RITE.VEXOTControl version 1.10.51

### PR DESCRIPTION
**Pull request by AndreiBee**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/171849)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Programs\VEXOTControl\VEXOTControl.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 895.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-09-09 11:10:32.036 [FAIL] Installer failed security check. Url: https://github.com/AndreiBee/VEXOTControl/releases/download/v1.10.51/VEXOTControlInstaller_v1.10.51.exe Result: 0x80004005

(Automated response - build 895.)

File: manifests/r/RITE/VEXOTControl/1.10.51/RITE.VEXOTControl.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- silentWithProgress
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /SILENT
+  Custom: /SUPPRESSMSGBOXES /NORESTART
+UpgradeBehavior: install
+Commands:
+- vexotcontrol
+ReleaseDate: 2024-09-04
+Installers:
```
Comment (stephengillie) [inline manifests/r/RITE/VEXOTControl/1.10.51/RITE.VEXOTControl.installer.yaml:20]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @AndreiBee,

It appears that this package has a dependency on `VCRedist`:
image

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**